CONTRIBUTING.md

```diff
@@ -14,6 +14,62 @@ Thanks for taking the plunge!
   (`Pkg.add(name="DataFrames", rev="main")`) is a good gut check and can streamline the process,
   along with including the first two lines of output from `versioninfo()`
 
+## Modifying an existing docstring in `src/`
+
+All docstrings are written inline above the methods or types they are associated with and can
+be found by clicking on the `source` link that appears below each docstring in the HTML file.
+The steps needed to make a change to an existing docstring are listed below:
+
+* Find the docstring in `src/`;
+* Update the text in the docstring;
+* run `make docs` from the root directory;
+* check the output in `doc/_build/html/` to make sure the changes are correct;
+* commit your changes and open a pull request.
+
+## Adding a new docstring to `src/`
+
+The steps required to add a new docstring are listed below:
+* find a suitable definition in `src/` that the docstring will be most applicable to;
+* add a doctring above the definition;
+* find a suitable `@docs` code block in the `docs/src/lib/functions.md` file where you 
+  would like the docstring to appear;
+* add the name of the definition to the `@docs` code block. For example, with a docstring
+  added to a function `bar`.
+```
+"..."
+function bar(args...)
+    # ...
+end
+```
+
+you would add the name `bar` to a `@docs` block in `docs/src/lib/functions.md`
+````
+```@docs
+foo
+bar # <-- Added this one.
+baz
+```
+````
+
+* run `julia make.jl` from the `/docs` directory;
+* check the output in `docs/_build/html` to make sure the changes are correct;
+* commit your changes and open a pull request.
+
+## Doctests
+
+Examples written within docstrings can be used as test cases known as *doctests* by annotating code 
+blocks with `jldoctest`.
+````
+```jldoctest
+julia> uppercase("Docstring test")
+"DOCSTRING TEST"
+```
+````
+A doctest needs to match an interactive REPL including the `julia>` prompt. To run doctests you need
+to make `doctest=true` inside the `docs/make.jl` directory and run `julia make.jl` from the `/docs` directory.
```

Comment:
as commented `doctest=true` is already set so I think there is no need to mention that.